# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/mac/moz.build
# Commit: 9bc12053de5e
# Full Hash: 9bc12053de5e6b071d41c81d82859d1567c80c42
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2020-07-03 03:56:55
# Regressor Bug: 1649575
# File Overlap Count: 2
# Description:
#   Bug 1649575 - Part 4: Introduce GeckoTextMarker and GeckoTextMarkerRange. r=morgan
#   
#   These classes do a few things:
#   1. They implement and abstract common text operations, for example:
#    (a) comparing two markers to know which one precedes the other.
# ==============================================================================

diff -r 0441d1553b6b -r 9bc12053de5e accessible/mac/moz.build
--- a/accessible/mac/moz.build	Thu Jul 02 21:33:07 2020 +0000
+++ b/accessible/mac/moz.build	Thu Jul 02 21:35:05 2020 +0000
@@ -16,6 +16,7 @@
 UNIFIED_SOURCES += [
     'AccessibleWrap.mm',
     'DocAccessibleWrap.mm',
+    'GeckoTextMarker.mm',
     'MacUtils.mm',
     'MOXAccessibleBase.mm',
     'MOXMathAccessibles.mm',